web/Gruntfile.js

```diff
@@ -40,8 +45,12 @@ module.exports = function (grunt) {
                     '<%= config.srcScriptsDir %>/**/*.es.js'
                 ],
                 tasks: [
-                    'compilejs'
+                    'build'
                 ]
+            },
+            ngtemplates: {
+                files: ['app/templates/**/*.html'],
+                tasks: ['build']
```

Comment:
Also I need to `build` because I browserify so I need that file for testing anyway